wp-content/themes/pawar2018/js/scripts/app.js

```diff
@@ -17,6 +17,13 @@
   for (i = 0; i < accords.length; ++i) {
     let accordContent = accords[i].querySelector(".accordion-content");
     let accordTitle = accords[i].querySelector(".accordion-title");
+    if (window.location.hash) {
+      var hash = window.location.hash.substring(1);
+      if (hash === accordTitle.innerText.toLowerCase().split(' ').join('-')) {
```

Comment:
Ugh I am a dolt! I didn't even see I had put `name` in there already for this. I'll be fixing it.